### ISSUE TITLE
Python library compilation failure on OSX Yosemite

### ISSUE THREAD

**Issue by jeremyherbert** [libigl/libigl#222]
Hi,

I'm trying to compile the python library on yosemite, but I get stuck at this error:

```
In file included from /Users/jeremy/code/libigl/python/py_igl.cpp:47:
In file included from /Users/jeremy/code/libigl/python/../include/igl/find_cross_field_singularities.h:55:
/Users/jeremy/code/libigl/python/../include/igl/find_cross_field_singularities.cpp:77:3: error: no matching function for call to 'cross_field_missmatch'
  igl::cross_field_missmatch<DerivedV, DerivedF>(V, F, PD1, PD2, isCombed, Handle_MMatch);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/jeremy/code/libigl/python/py_igl/py_find_cross_field_singularities.cpp:25:15: note: in instantiation of function template specialization
      'igl::find_cross_field_singularities<Eigen::Matrix<double, -1, -1, 0, -1, -1>, Eigen::Matrix<int, -1, -1, 0, -1, -1>, Eigen::Matrix<int, -1, -1, 0, -1, -1> >'
      requested here
  return igl::find_cross_field_singularities(V,F,PD1,PD2,isSingularity,singularityIndex,isCombed);
              ^
/Users/jeremy/code/libigl/python/../include/igl/cross_field_missmatch.cpp:114:22: note: candidate function [with DerivedV = Eigen::Matrix<double, -1, -1, 0, -1, -1>,
      DerivedF = Eigen::Matrix<int, -1, -1, 0, -1, -1>] not viable: no known conversion from 'Eigen::Matrix<typename Matrix<int, -1, -1, 0, -1, -1>::Scalar,
      Eigen::Dynamic, 3>' to 'Eigen::PlainObjectBase<Matrix<int, -1, -1, 0, -1, -1> > &' for 6th argument
IGL_INLINE void igl::cross_field_missmatch(const Eigen::PlainObjectBase<DerivedV> &V,
                     ^
```

Is this a known problem ?

**Comment by cubic23** [comment]
I got the same problem on Ubuntu 15.10. As a temporary fix to make the compilation work you can comment out the mentioned line.

**Comment by danielepanozzo** [comment]
Thank you for the report, it is fixed in #2a7b0b9. I am closing this issue.

The python wrappers are still very experimental: I would be very interested on your feedback if you are already using them.
